Flaky homematicip cloud tests

<!-- READ THIS FIRST:
  - If you need additional help with this template, please refer to https://www.home-assistant.io/help/reporting_issues/
  - Make sure you are running the latest version of Home Assistant before reporting an issue: https://github.com/home-assistant/home-assistant/releases
  - Do not report issues for integrations if you are using custom components or integrations.
  - Provide as many details as possible. Paste logs, configuration samples and code into the backticks.
  DO NOT DELETE ANY TEXT from this template! Otherwise, your issue may be closed without comment.
-->
## The problem
<!-- 
  Describe the issue you are experiencing here to communicate to the
  maintainers. Tell us what you were trying to do and what happened instead.
-->
2 flaky tests from homematicip cloud integration seem to exist on dev branch.

## Environment
<!--
  Provide details about the versions you are using, which helps us to reproduce
  and find the issue quicker. Version information is found in the
  Home Assistant frontend: Developer tools -> Info.
-->

- Home Assistant release with the issue: 0.108 dev
- Last working Home Assistant release (if known): 
- Operating environment (Hass.io/Docker/Windows/etc.): Azure pipelines
- Integration causing this issue: homematicip_cloud
- Link to integration documentation on our website: https://www.home-assistant.io/integrations/homematicip_cloud/

## Problem-relevant `configuration.yaml`
<!--
  An example configuration that caused the problem for you. Fill this out even
  if it seems unimportant to you. Please be sure to remove personal information
  like passwords, private URLs and other credentials.
-->

```yaml

```

## Traceback/Error logs
<!--
  If you come across any trace or error logs, please provide them.
-->

```txt
2020-03-13T23:27:23.1650516Z ##[error]2 test(s) failed, 8245 test(s) collected.
2020-03-13T23:27:23.1744153Z Reading code coverage summary from '/home/vsts/work/1/s/coverage.xml'
2020-03-13T23:27:23.5070468Z 
2020-03-13T23:27:23.5071324Z ==================================== ERRORS ====================================
2020-03-13T23:27:23.5072019Z _ ERROR at teardown of test_config_with_accesspoint_passed_to_config_entry[pyloop] _
2020-03-13T23:27:23.5072945Z [gw0] linux -- Python 3.7.0 /__w/1/s/venv/bin/python
2020-03-13T23:27:23.5073195Z 
2020-03-13T23:27:23.5073460Z loop = <_UnixSelectorEventLoop running=False closed=True debug=False>
2020-03-13T23:27:23.5073767Z hass_storage = {}
2020-03-13T23:27:23.5073941Z 
2020-03-13T23:27:23.5074147Z     @pytest.fixture
2020-03-13T23:27:23.5074386Z     def hass(loop, hass_storage):
2020-03-13T23:27:23.5074693Z         """Fixture to provide a test instance of Home Assistant."""
2020-03-13T23:27:23.5075069Z         hass = loop.run_until_complete(async_test_home_assistant(loop))
2020-03-13T23:27:23.5075343Z     
2020-03-13T23:27:23.5075557Z         yield hass
2020-03-13T23:27:23.5075752Z     
2020-03-13T23:27:23.5076018Z >       loop.run_until_complete(hass.async_stop(force=True))
2020-03-13T23:27:23.5076259Z 
2020-03-13T23:27:23.5076463Z tests/conftest.py:86: 
2020-03-13T23:27:23.5076799Z _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
2020-03-13T23:27:23.5077193Z /usr/local/lib/python3.7/asyncio/base_events.py:555: in run_until_complete
2020-03-13T23:27:23.5077517Z     self.run_forever()
2020-03-13T23:27:23.5077815Z /usr/local/lib/python3.7/asyncio/base_events.py:523: in run_forever
2020-03-13T23:27:23.5078114Z     self._run_once()
2020-03-13T23:27:23.5078421Z /usr/local/lib/python3.7/asyncio/base_events.py:1722: in _run_once
2020-03-13T23:27:23.5078753Z     event_list = self._selector.select(timeout)
2020-03-13T23:27:23.5079112Z _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
2020-03-13T23:27:23.5079376Z 
2020-03-13T23:27:23.5079643Z self = <selectors.EpollSelector object at 0x7fb20c3dfa58>, timeout = 2.996
2020-03-13T23:27:23.5079915Z 
2020-03-13T23:27:23.5080126Z     def select(self, timeout=None):
2020-03-13T23:27:23.5080393Z         if timeout is None:
2020-03-13T23:27:23.5080782Z             timeout = -1
2020-03-13T23:27:23.5081034Z         elif timeout <= 0:
2020-03-13T23:27:23.5081284Z             timeout = 0
2020-03-13T23:27:23.5081503Z         else:
2020-03-13T23:27:23.5081810Z             # epoll_wait() has a resolution of 1 millisecond, round away
2020-03-13T23:27:23.5082166Z             # from zero to wait *at least* timeout seconds.
2020-03-13T23:27:23.5082839Z             timeout = math.ceil(timeout * 1e3) * 1e-3
2020-03-13T23:27:23.5083104Z     
2020-03-13T23:27:23.5083369Z         # epoll_wait() expects `maxevents` to be greater than zero;
2020-03-13T23:27:23.5083739Z         # we want to make sure that `select()` can be called when no
2020-03-13T23:27:23.5084200Z         # FD is registered.
2020-03-13T23:27:23.5084608Z         max_ev = max(len(self._fd_to_key), 1)
2020-03-13T23:27:23.5085028Z     
2020-03-13T23:27:23.5085240Z         ready = []
2020-03-13T23:27:23.5085448Z         try:
2020-03-13T23:27:23.5085719Z >           fd_event_list = self._selector.poll(timeout, max_ev)
2020-03-13T23:27:23.5086048Z E           Failed: Timeout >9.0s
2020-03-13T23:27:23.5086257Z 
2020-03-13T23:27:23.5086511Z /usr/local/lib/python3.7/selectors.py:468: Failed
2020-03-13T23:27:23.5087070Z ---------------------------- Captured stderr setup -----------------------------
2020-03-13T23:27:23.5087437Z DEBUG:asyncio:Using selector: EpollSelector
2020-03-13T23:27:23.5087966Z ------------------------------ Captured log setup ------------------------------
2020-03-13T23:27:23.5088375Z DEBUG    asyncio:selector_events.py:64 Using selector: EpollSelector
2020-03-13T23:27:23.5088943Z ----------------------------- Captured stderr call -----------------------------
2020-03-13T23:27:23.5089336Z INFO:homeassistant.loader:Loaded test from custom_components.test
2020-03-13T23:27:23.5089745Z INFO:homeassistant.loader:Loaded test_package from custom_components.test_package
2020-03-13T23:27:23.5090184Z INFO:homeassistant.loader:Loaded homematicip_cloud from homeassistant.components.homematicip_cloud
2020-03-13T23:27:23.5090680Z INFO:homeassistant.setup:Setting up homematicip_cloud
2020-03-13T23:27:23.5091051Z INFO:homeassistant.setup:Setup of domain homematicip_cloud took 0.0 seconds.
2020-03-13T23:27:23.5091512Z DEBUG:homeassistant.core:Bus:Handling <Event component_loaded[L]: component=homematicip_cloud>
2020-03-13T23:27:23.5091901Z INFO:.:Imported authentication for ABC123
2020-03-13T23:27:23.5092432Z ------------------------------ Captured log call -------------------------------
2020-03-13T23:27:23.5093028Z INFO     homeassistant.loader:loader.py:207 Loaded test from custom_components.test
2020-03-13T23:27:23.5093496Z INFO     homeassistant.loader:loader.py:207 Loaded test_package from custom_components.test_package
2020-03-13T23:27:23.5094010Z INFO     homeassistant.loader:loader.py:207 Loaded homematicip_cloud from homeassistant.components.homematicip_cloud
2020-03-13T23:27:23.5094495Z INFO     homeassistant.setup:setup.py:153 Setting up homematicip_cloud
2020-03-13T23:27:23.5094927Z INFO     homeassistant.setup:setup.py:187 Setup of domain homematicip_cloud took 0.0 seconds.
2020-03-13T23:27:23.5095421Z DEBUG    homeassistant.core:core.py:578 Bus:Handling <Event component_loaded[L]: component=homematicip_cloud>
2020-03-13T23:27:23.5095888Z INFO     .:config_flow.py:93 Imported authentication for ABC123
2020-03-13T23:27:23.5096447Z --------------------------- Captured stderr teardown ---------------------------
2020-03-13T23:27:23.5096858Z DEBUG:homeassistant.core:Bus:Handling <Event homeassistant_stop[L]>
2020-03-13T23:27:23.5097253Z ERROR:homeassistant.core:Error doing job: Task exception was never retrieved
2020-03-13T23:27:23.5097606Z Traceback (most recent call last):
2020-03-13T23:27:23.5097911Z   File "/__w/1/s/homeassistant/core.py", line 1198, in async_call
2020-03-13T23:27:23.5098429Z     raise ServiceNotFound(domain, service) from None
2020-03-13T23:27:23.5098873Z homeassistant.exceptions.ServiceNotFound: Unable to find service persistent_notification/dismiss
2020-03-13T23:27:23.5099502Z DEBUG:homematicip.aio.connection:Connection timed out or another error occurred https://lookup.homematic.com:48335/getHost
2020-03-13T23:27:23.5099929Z 
2020-03-13T23:27:23.5100239Z +++++++++++++++++++++++++++++++++++ Timeout ++++++++++++++++++++++++++++++++++++
2020-03-13T23:27:23.5100551Z 
2020-03-13T23:27:23.5100862Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_8 (140402195683072) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5101304Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5101651Z     self._bootstrap_inner()
2020-03-13T23:27:23.5102008Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5102360Z     self.run()
2020-03-13T23:27:23.5102678Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5103136Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5103684Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-03-13T23:27:23.5107476Z     work_item = work_queue.get(block=True)
2020-03-13T23:27:23.5107884Z 
2020-03-13T23:27:23.5109973Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_7 (140402204075776) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5110551Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5110930Z     self._bootstrap_inner()
2020-03-13T23:27:23.5111335Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5111698Z     self.run()
2020-03-13T23:27:23.5112051Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5112815Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5113295Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-03-13T23:27:23.5113744Z     work_item = work_queue.get(block=True)
2020-03-13T23:27:23.5114016Z 
2020-03-13T23:27:23.5114379Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_6 (140402723636992) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5115009Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5115400Z     self._bootstrap_inner()
2020-03-13T23:27:23.5115789Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5116170Z     self.run()
2020-03-13T23:27:23.5116515Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5116914Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5117359Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-03-13T23:27:23.5117784Z     work_item = work_queue.get(block=True)
2020-03-13T23:27:23.5118059Z 
2020-03-13T23:27:23.5118399Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_5 (140402740422400) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5118864Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5119309Z     self._bootstrap_inner()
2020-03-13T23:27:23.5119621Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5119936Z     self.run()
2020-03-13T23:27:23.5120402Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5120783Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5121182Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-03-13T23:27:23.5121588Z     work_item = work_queue.get(block=True)
2020-03-13T23:27:23.5121830Z 
2020-03-13T23:27:23.5122146Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_4 (140402748815104) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5122593Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5122943Z     self._bootstrap_inner()
2020-03-13T23:27:23.5123312Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5123654Z     self.run()
2020-03-13T23:27:23.5124254Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5124638Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5124992Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 80, in _worker
2020-03-13T23:27:23.5125368Z     work_item.run()
2020-03-13T23:27:23.5125719Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
2020-03-13T23:27:23.5126129Z     result = self.fn(*self.args, **self.kwargs)
2020-03-13T23:27:23.5126508Z   File "/usr/local/lib/python3.7/socket.py", line 748, in getaddrinfo
2020-03-13T23:27:23.5126948Z     for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
2020-03-13T23:27:23.5127245Z 
2020-03-13T23:27:23.5127559Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_3 (140402732029696) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5128005Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5128447Z     self._bootstrap_inner()
2020-03-13T23:27:23.5128820Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5129225Z     self.run()
2020-03-13T23:27:23.5129567Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5129934Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5130329Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-03-13T23:27:23.5130733Z     work_item = work_queue.get(block=True)
2020-03-13T23:27:23.5130974Z 
2020-03-13T23:27:23.5131303Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_2 (140402176800512) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5131740Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5132098Z     self._bootstrap_inner()
2020-03-13T23:27:23.5132458Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5132795Z     self.run()
2020-03-13T23:27:23.5133130Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5133494Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5133905Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-03-13T23:27:23.5134362Z     work_item = work_queue.get(block=True)
2020-03-13T23:27:23.5134617Z 
2020-03-13T23:27:23.5135043Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_1 (140402715244288) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5135424Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5135744Z     self._bootstrap_inner()
2020-03-13T23:27:23.5136060Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5136369Z     self.run()
2020-03-13T23:27:23.5136646Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5136979Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5137329Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-03-13T23:27:23.5137677Z     work_item = work_queue.get(block=True)
2020-03-13T23:27:23.5137902Z 
2020-03-13T23:27:23.5138134Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_0 (140402212468480) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5138531Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5138837Z     self._bootstrap_inner()
2020-03-13T23:27:23.5139162Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5139735Z     self.run()
2020-03-13T23:27:23.5140008Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5140302Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5140603Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-03-13T23:27:23.5140916Z     work_item = work_queue.get(block=True)
2020-03-13T23:27:23.5141083Z 
2020-03-13T23:27:23.5141679Z ~~~~~~~~~~~~~~~~~~ Stack of zeroconf-Reaper (140402160015104) ~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5142046Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5142308Z     self._bootstrap_inner()
2020-03-13T23:27:23.5142592Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5142846Z     self.run()
2020-03-13T23:27:23.5143360Z   File "/__w/1/s/venv/lib/python3.7/site-packages/zeroconf/__init__.py", line 1305, in run
2020-03-13T23:27:23.5143665Z     self.zc.wait(10 * 1000)
2020-03-13T23:27:23.5144135Z   File "/__w/1/s/venv/lib/python3.7/site-packages/zeroconf/__init__.py", line 2178, in wait
2020-03-13T23:27:23.5144459Z     self.condition.wait(timeout / 1000.0)
2020-03-13T23:27:23.5144733Z   File "/usr/local/lib/python3.7/threading.py", line 300, in wait
2020-03-13T23:27:23.5145020Z     gotit = waiter.acquire(True, timeout)
2020-03-13T23:27:23.5145187Z 
2020-03-13T23:27:23.5145593Z ~~~~~~~~~~~~~~~~~~ Stack of zeroconf-Engine (140402168407808) ~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5146013Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5146290Z     self._bootstrap_inner()
2020-03-13T23:27:23.5146560Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5146816Z     self.run()
2020-03-13T23:27:23.5147274Z   File "/__w/1/s/venv/lib/python3.7/site-packages/zeroconf/__init__.py", line 1226, in run
2020-03-13T23:27:23.5147649Z     rr, wr, er = select.select(cast(Sequence[Any], rs), [], [], self.timeout)
2020-03-13T23:27:23.5147883Z 
2020-03-13T23:27:23.5148116Z ~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (140404147357440) ~~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5148684Z   File "/__w/1/s/venv/lib/python3.7/site-packages/execnet/gateway_base.py", line 285, in _perform_spawn
2020-03-13T23:27:23.5148986Z     reply.run()
2020-03-13T23:27:23.5149425Z   File "/__w/1/s/venv/lib/python3.7/site-packages/execnet/gateway_base.py", line 220, in run
2020-03-13T23:27:23.5149764Z     self._result = func(*args, **kwargs)
2020-03-13T23:27:23.5150269Z   File "/__w/1/s/venv/lib/python3.7/site-packages/execnet/gateway_base.py", line 967, in _thread_receiver
2020-03-13T23:27:23.5150609Z     msg = Message.from_io(io)
2020-03-13T23:27:23.5151135Z   File "/__w/1/s/venv/lib/python3.7/site-packages/execnet/gateway_base.py", line 432, in from_io
2020-03-13T23:27:23.5151500Z     header = io.read(9)  # type 1, channel 4, payload 4
2020-03-13T23:27:23.5152005Z   File "/__w/1/s/venv/lib/python3.7/site-packages/execnet/gateway_base.py", line 400, in read
2020-03-13T23:27:23.5152479Z     data = self._read(numbytes - len(buf))
2020-03-13T23:27:23.5152655Z 
2020-03-13T23:27:23.5152885Z +++++++++++++++++++++++++++++++++++ Timeout ++++++++++++++++++++++++++++++++++++
2020-03-13T23:27:23.5153396Z ---------------------------- Captured log teardown -----------------------------
2020-03-13T23:27:23.5153777Z DEBUG    homeassistant.core:core.py:578 Bus:Handling <Event homeassistant_stop[L]>
2020-03-13T23:27:23.5154201Z ERROR    homeassistant.core:core.py:143 Error doing job: Task exception was never retrieved
2020-03-13T23:27:23.5154528Z Traceback (most recent call last):
2020-03-13T23:27:23.5154806Z   File "/__w/1/s/homeassistant/core.py", line 1198, in async_call
2020-03-13T23:27:23.5155110Z     raise ServiceNotFound(domain, service) from None
2020-03-13T23:27:23.5155467Z homeassistant.exceptions.ServiceNotFound: Unable to find service persistent_notification/dismiss
2020-03-13T23:27:23.5155997Z DEBUG    homematicip.aio.connection:connection.py:101 Connection timed out or another error occurred https://lookup.homematic.com:48335/getHost
2020-03-13T23:27:23.5156506Z =================================== FAILURES ===================================
2020-03-13T23:27:23.5156850Z ___________________________ test_flow_works[pyloop] ____________________________
2020-03-13T23:27:23.5157313Z [gw0] linux -- Python 3.7.0 /__w/1/s/venv/bin/python
2020-03-13T23:27:23.5157507Z 
2020-03-13T23:27:23.5157693Z pyfuncitem = <Function test_flow_works[pyloop]>
2020-03-13T23:27:23.5157882Z 
2020-03-13T23:27:23.5158100Z     def pytest_pyfunc_call(pyfuncitem):  # type: ignore
2020-03-13T23:27:23.5158333Z         """
2020-03-13T23:27:23.5158589Z         Run coroutines in an event loop instead of a normal function call.
2020-03-13T23:27:23.5158834Z         """
2020-03-13T23:27:23.5159215Z         fast = pyfuncitem.config.getoption("--aiohttp-fast")
2020-03-13T23:27:23.5159519Z         if asyncio.iscoroutinefunction(pyfuncitem.function):
2020-03-13T23:27:23.5159985Z             existing_loop = pyfuncitem.funcargs.get('proactor_loop')\
2020-03-13T23:27:23.5160432Z                 or pyfuncitem.funcargs.get('loop', None)
2020-03-13T23:27:23.5160706Z             with _runtime_warning_context():
2020-03-13T23:27:23.5161021Z                 with _passthrough_loop_context(existing_loop, fast=fast) as _loop:
2020-03-13T23:27:23.5161363Z                     testargs = {arg: pyfuncitem.funcargs[arg]
2020-03-13T23:27:23.5161703Z                                 for arg in pyfuncitem._fixtureinfo.argnames}
2020-03-13T23:27:23.5162087Z >                   _loop.run_until_complete(pyfuncitem.obj(**testargs))
2020-03-13T23:27:23.5162310Z 
2020-03-13T23:27:23.5162686Z venv/lib/python3.7/site-packages/aiohttp/pytest_plugin.py:170: 
2020-03-13T23:27:23.5163039Z _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
2020-03-13T23:27:23.5163407Z /usr/local/lib/python3.7/asyncio/base_events.py:555: in run_until_complete
2020-03-13T23:27:23.5163673Z     self.run_forever()
2020-03-13T23:27:23.5163942Z /usr/local/lib/python3.7/asyncio/base_events.py:523: in run_forever
2020-03-13T23:27:23.5164406Z     self._run_once()
2020-03-13T23:27:23.5164681Z /usr/local/lib/python3.7/asyncio/base_events.py:1722: in _run_once
2020-03-13T23:27:23.5164997Z     event_list = self._selector.select(timeout)
2020-03-13T23:27:23.5165317Z _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
2020-03-13T23:27:23.5165539Z 
2020-03-13T23:27:23.5165764Z self = <selectors.EpollSelector object at 0x7fb20c82f630>, timeout = 2.998
2020-03-13T23:27:23.5165994Z 
2020-03-13T23:27:23.5166163Z     def select(self, timeout=None):
2020-03-13T23:27:23.5166372Z         if timeout is None:
2020-03-13T23:27:23.5167003Z             timeout = -1
2020-03-13T23:27:23.5167239Z         elif timeout <= 0:
2020-03-13T23:27:23.5167475Z             timeout = 0
2020-03-13T23:27:23.5167675Z         else:
2020-03-13T23:27:23.5167966Z             # epoll_wait() has a resolution of 1 millisecond, round away
2020-03-13T23:27:23.5168322Z             # from zero to wait *at least* timeout seconds.
2020-03-13T23:27:23.5168807Z             timeout = math.ceil(timeout * 1e3) * 1e-3
2020-03-13T23:27:23.5169069Z     
2020-03-13T23:27:23.5169326Z         # epoll_wait() expects `maxevents` to be greater than zero;
2020-03-13T23:27:23.5169715Z         # we want to make sure that `select()` can be called when no
2020-03-13T23:27:23.5170010Z         # FD is registered.
2020-03-13T23:27:23.5170281Z         max_ev = max(len(self._fd_to_key), 1)
2020-03-13T23:27:23.5170504Z     
2020-03-13T23:27:23.5170678Z         ready = []
2020-03-13T23:27:23.5170884Z         try:
2020-03-13T23:27:23.5171146Z >           fd_event_list = self._selector.poll(timeout, max_ev)
2020-03-13T23:27:23.5171472Z E           Failed: Timeout >9.0s
2020-03-13T23:27:23.5171661Z 
2020-03-13T23:27:23.5171886Z /usr/local/lib/python3.7/selectors.py:468: Failed
2020-03-13T23:27:23.5172442Z ---------------------------- Captured stderr setup -----------------------------
2020-03-13T23:27:23.5172809Z DEBUG:asyncio:Using selector: EpollSelector
2020-03-13T23:27:23.5173353Z ------------------------------ Captured log setup ------------------------------
2020-03-13T23:27:23.5173770Z DEBUG    asyncio:selector_events.py:64 Using selector: EpollSelector
2020-03-13T23:27:23.5174368Z ----------------------------- Captured stderr call -----------------------------
2020-03-13T23:27:23.5174768Z INFO:homeassistant.loader:Loaded test from custom_components.test
2020-03-13T23:27:23.5175187Z INFO:homeassistant.loader:Loaded test_package from custom_components.test_package
2020-03-13T23:27:23.5175642Z INFO:homeassistant.loader:Loaded homematicip_cloud from homeassistant.components.homematicip_cloud
2020-03-13T23:27:23.5176204Z DEBUG:homematicip.aio.connection:Connection timed out or another error occurred https://lookup.homematic.com:48335/getHost
2020-03-13T23:27:23.5176593Z 
2020-03-13T23:27:23.5176859Z +++++++++++++++++++++++++++++++++++ Timeout ++++++++++++++++++++++++++++++++++++
2020-03-13T23:27:23.5177127Z 
2020-03-13T23:27:23.5177391Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_8 (140402195683072) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5177775Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5178091Z     self._bootstrap_inner()
2020-03-13T23:27:23.5178400Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5178703Z     self.run()
2020-03-13T23:27:23.5178969Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5179367Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5179718Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-03-13T23:27:23.5180079Z     work_item = work_queue.get(block=True)
2020-03-13T23:27:23.5180271Z 
2020-03-13T23:27:23.5180535Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_7 (140402204075776) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5180932Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5181230Z     self._bootstrap_inner()
2020-03-13T23:27:23.5181553Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5181841Z     self.run()
2020-03-13T23:27:23.5182120Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5182439Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5182784Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-03-13T23:27:23.5183251Z     work_item = work_queue.get(block=True)
2020-03-13T23:27:23.5183419Z 
2020-03-13T23:27:23.5183661Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_6 (140402212468480) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5184037Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5184312Z     self._bootstrap_inner()
2020-03-13T23:27:23.5184587Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5184835Z     self.run()
2020-03-13T23:27:23.5185081Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5185358Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5185671Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-03-13T23:27:23.5185971Z     work_item = work_queue.get(block=True)
2020-03-13T23:27:23.5186136Z 
2020-03-13T23:27:23.5186379Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_5 (140402723636992) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5186715Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5186989Z     self._bootstrap_inner()
2020-03-13T23:27:23.5187258Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5187525Z     self.run()
2020-03-13T23:27:23.5187757Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5188046Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5188349Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-03-13T23:27:23.5188647Z     work_item = work_queue.get(block=True)
2020-03-13T23:27:23.5188828Z 
2020-03-13T23:27:23.5189055Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_4 (140402748815104) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5189401Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5189663Z     self._bootstrap_inner()
2020-03-13T23:27:23.5189928Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5190195Z     self.run()
2020-03-13T23:27:23.5190426Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5190717Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5191017Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-03-13T23:27:23.5191329Z     work_item = work_queue.get(block=True)
2020-03-13T23:27:23.5191497Z 
2020-03-13T23:27:23.5191724Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_3 (140401809815296) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5192070Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5192330Z     self._bootstrap_inner()
2020-03-13T23:27:23.5193096Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5193396Z     self.run()
2020-03-13T23:27:23.5193675Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5193991Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5194406Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-03-13T23:27:23.5194752Z     work_item = work_queue.get(block=True)
2020-03-13T23:27:23.5194945Z 
2020-03-13T23:27:23.5195222Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_2 (140402715244288) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5195606Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5195922Z     self._bootstrap_inner()
2020-03-13T23:27:23.5196232Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5196531Z     self.run()
2020-03-13T23:27:23.5196797Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5197114Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5197476Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 80, in _worker
2020-03-13T23:27:23.5197861Z     work_item.run()
2020-03-13T23:27:23.5198229Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
2020-03-13T23:27:23.5198791Z     result = self.fn(*self.args, **self.kwargs)
2020-03-13T23:27:23.5199138Z   File "/usr/local/lib/python3.7/socket.py", line 748, in getaddrinfo
2020-03-13T23:27:23.5199561Z     for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
2020-03-13T23:27:23.5199822Z 
2020-03-13T23:27:23.5200109Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_1 (140402732029696) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5200489Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5200810Z     self._bootstrap_inner()
2020-03-13T23:27:23.5201122Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5201431Z     self.run()
2020-03-13T23:27:23.5201707Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5202027Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5202387Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-03-13T23:27:23.5202733Z     work_item = work_queue.get(block=True)
2020-03-13T23:27:23.5202958Z 
2020-03-13T23:27:23.5203237Z ~~~~~~~~~~~~~~~~~~~ Stack of SyncWorker_0 (140402740422400) ~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5203631Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5203936Z     self._bootstrap_inner()
2020-03-13T23:27:23.5204308Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5204620Z     self.run()
2020-03-13T23:27:23.5204894Z   File "/usr/local/lib/python3.7/threading.py", line 865, in run
2020-03-13T23:27:23.5205623Z     self._target(*self._args, **self._kwargs)
2020-03-13T23:27:23.5206116Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 78, in _worker
2020-03-13T23:27:23.5206696Z     work_item = work_queue.get(block=True)
2020-03-13T23:27:23.5206935Z 
2020-03-13T23:27:23.5207535Z ~~~~~~~~~~~~~~~~~~ Stack of zeroconf-Reaper (140402160015104) ~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5208014Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5208369Z     self._bootstrap_inner()
2020-03-13T23:27:23.5208738Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5209076Z     self.run()
2020-03-13T23:27:23.5209649Z   File "/__w/1/s/venv/lib/python3.7/site-packages/zeroconf/__init__.py", line 1305, in run
2020-03-13T23:27:23.5210051Z     self.zc.wait(10 * 1000)
2020-03-13T23:27:23.5210632Z   File "/__w/1/s/venv/lib/python3.7/site-packages/zeroconf/__init__.py", line 2178, in wait
2020-03-13T23:27:23.5211070Z     self.condition.wait(timeout / 1000.0)
2020-03-13T23:27:23.5211435Z   File "/usr/local/lib/python3.7/threading.py", line 300, in wait
2020-03-13T23:27:23.5211816Z     gotit = waiter.acquire(True, timeout)
2020-03-13T23:27:23.5212053Z 
2020-03-13T23:27:23.5212573Z ~~~~~~~~~~~~~~~~~~ Stack of zeroconf-Engine (140402168407808) ~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5213109Z   File "/usr/local/lib/python3.7/threading.py", line 885, in _bootstrap
2020-03-13T23:27:23.5213460Z     self._bootstrap_inner()
2020-03-13T23:27:23.5213832Z   File "/usr/local/lib/python3.7/threading.py", line 917, in _bootstrap_inner
2020-03-13T23:27:23.5214170Z     self.run()
2020-03-13T23:27:23.5214743Z   File "/__w/1/s/venv/lib/python3.7/site-packages/zeroconf/__init__.py", line 1226, in run
2020-03-13T23:27:23.5215225Z     rr, wr, er = select.select(cast(Sequence[Any], rs), [], [], self.timeout)
2020-03-13T23:27:23.5215542Z 
2020-03-13T23:27:23.5215856Z ~~~~~~~~~~~~~~~~~~~~~ Stack of <unknown> (140404147357440) ~~~~~~~~~~~~~~~~~~~~~
2020-03-13T23:27:23.5216548Z   File "/__w/1/s/venv/lib/python3.7/site-packages/execnet/gateway_base.py", line 285, in _perform_spawn
2020-03-13T23:27:23.5216966Z     reply.run()
2020-03-13T23:27:23.5217529Z   File "/__w/1/s/venv/lib/python3.7/site-packages/execnet/gateway_base.py", line 220, in run
2020-03-13T23:27:23.5217969Z     self._result = func(*args, **kwargs)
2020-03-13T23:27:23.5218608Z   File "/__w/1/s/venv/lib/python3.7/site-packages/execnet/gateway_base.py", line 967, in _thread_receiver
2020-03-13T23:27:23.5219127Z     msg = Message.from_io(io)
2020-03-13T23:27:23.5220049Z   File "/__w/1/s/venv/lib/python3.7/site-packages/execnet/gateway_base.py", line 432, in from_io
2020-03-13T23:27:23.5220485Z     header = io.read(9)  # type 1, channel 4, payload 4
2020-03-13T23:27:23.5221236Z   File "/__w/1/s/venv/lib/python3.7/site-packages/execnet/gateway_base.py", line 400, in read
2020-03-13T23:27:23.5221830Z     data = self._read(numbytes - len(buf))
2020-03-13T23:27:23.5222084Z 
2020-03-13T23:27:23.5222395Z +++++++++++++++++++++++++++++++++++ Timeout ++++++++++++++++++++++++++++++++++++
2020-03-13T23:27:23.5223039Z ------------------------------ Captured log call -------------------------------
2020-03-13T23:27:23.5223524Z INFO     homeassistant.loader:loader.py:207 Loaded test from custom_components.test
2020-03-13T23:27:23.5224115Z INFO     homeassistant.loader:loader.py:207 Loaded test_package from custom_components.test_package
2020-03-13T23:27:23.5224692Z INFO     homeassistant.loader:loader.py:207 Loaded homematicip_cloud from homeassistant.components.homematicip_cloud
2020-03-13T23:27:23.5225401Z DEBUG    homematicip.aio.connection:connection.py:101 Connection timed out or another error occurred https://lookup.homematic.com:48335/getHost
2020-03-13T23:27:23.5226204Z --------------------------- Captured stderr teardown ---------------------------
2020-03-13T23:27:23.5226676Z DEBUG:homeassistant.core:Bus:Handling <Event homeassistant_stop[L]>
2020-03-13T23:27:23.5227109Z DEBUG:homeassistant.core:Bus:Handling <Event homeassistant_close[L]>
2020-03-13T23:27:23.5227578Z ERROR:homeassistant.core:Error doing job: Task was destroyed but it is pending!
2020-03-13T23:27:23.5228055Z ERROR:homeassistant.core:Error doing job: Future exception was never retrieved
2020-03-13T23:27:23.5228449Z Traceback (most recent call last):
2020-03-13T23:27:23.5228845Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
2020-03-13T23:27:23.5229240Z     result = self.fn(*self.args, **self.kwargs)
2020-03-13T23:27:23.5229632Z   File "/usr/local/lib/python3.7/socket.py", line 748, in getaddrinfo
2020-03-13T23:27:23.5230052Z     for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
2020-03-13T23:27:23.5230664Z socket.gaierror: [Errno -3] Temporary failure in name resolution
2020-03-13T23:27:23.5231292Z ---------------------------- Captured log teardown -----------------------------
2020-03-13T23:27:23.5231794Z DEBUG    homeassistant.core:core.py:578 Bus:Handling <Event homeassistant_stop[L]>
2020-03-13T23:27:23.5232289Z DEBUG    homeassistant.core:core.py:578 Bus:Handling <Event homeassistant_close[L]>
2020-03-13T23:27:23.5232821Z ERROR    homeassistant.core:core.py:143 Error doing job: Task was destroyed but it is pending!
2020-03-13T23:27:23.5233368Z ERROR    homeassistant.core:core.py:143 Error doing job: Future exception was never retrieved
2020-03-13T23:27:23.5233877Z Traceback (most recent call last):
2020-03-13T23:27:23.5234250Z   File "/usr/local/lib/python3.7/concurrent/futures/thread.py", line 57, in run
2020-03-13T23:27:23.5234645Z     result = self.fn(*self.args, **self.kwargs)
2020-03-13T23:27:23.5235037Z   File "/usr/local/lib/python3.7/socket.py", line 748, in getaddrinfo
2020-03-13T23:27:23.5235456Z     for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
2020-03-13T23:27:23.5236078Z socket.gaierror: [Errno -3] Temporary failure in name resolution
2020-03-13T23:27:23.5236519Z =============================== warnings summary ===============================
2020-03-13T23:27:23.5237119Z venv/lib/python3.7/site-packages/_pytest/junitxml.py:436
2020-03-13T23:27:23.5237968Z   /__w/1/s/venv/lib/python3.7/site-packages/_pytest/junitxml.py:436: PytestDeprecationWarning: The 'junit_family' default value will change to 'xunit2' in pytest 6.0.
2020-03-13T23:27:23.5238932Z   Add 'junit_family=xunit1' to your pytest.ini file to keep the current format in future versions of pytest and silence this warning.
2020-03-13T23:27:23.5239500Z     _issue_warning_captured(deprecated.JUNIT_XML_DEFAULT_FAMILY, config.hook, 2)
2020-03-13T23:27:23.5239881Z 
2020-03-13T23:27:23.5240344Z venv/lib/python3.7/site-packages/botocore/awsrequest.py:624
2020-03-13T23:27:23.5240938Z venv/lib/python3.7/site-packages/botocore/awsrequest.py:624
2020-03-13T23:27:23.5241907Z   /__w/1/s/venv/lib/python3.7/site-packages/botocore/awsrequest.py:624: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
2020-03-13T23:27:23.5242590Z     class HeadersDict(collections.MutableMapping):
2020-03-13T23:27:23.5242843Z 
2020-03-13T23:27:23.5243327Z venv/lib/python3.7/site-packages/hyper/http11/connection.py:13
2020-03-13T23:27:23.5243917Z venv/lib/python3.7/site-packages/hyper/http11/connection.py:13
2020-03-13T23:27:23.5244792Z venv/lib/python3.7/site-packages/hyper/http11/connection.py:13
2020-03-13T23:27:23.5245410Z venv/lib/python3.7/site-packages/hyper/http11/connection.py:13
2020-03-13T23:27:23.5246400Z   /__w/1/s/venv/lib/python3.7/site-packages/hyper/http11/connection.py:13: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
2020-03-13T23:27:23.5247085Z     from collections import Iterable, Mapping
2020-03-13T23:27:23.5247345Z 
2020-03-13T23:27:23.5247794Z venv/lib/python3.7/site-packages/hyperframe/flags.py:14
2020-03-13T23:27:23.5248349Z venv/lib/python3.7/site-packages/hyperframe/flags.py:14
2020-03-13T23:27:23.5249313Z   /__w/1/s/venv/lib/python3.7/site-packages/hyperframe/flags.py:14: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
2020-03-13T23:27:23.5249956Z     class Flags(collections.MutableSet):
2020-03-13T23:27:23.5250205Z 
2020-03-13T23:27:23.5250728Z venv/lib/python3.7/site-packages/google/protobuf/internal/api_implementation.py:154
2020-03-13T23:27:23.5251417Z venv/lib/python3.7/site-packages/google/protobuf/internal/api_implementation.py:154
2020-03-13T23:27:23.5252496Z   /__w/1/s/venv/lib/python3.7/site-packages/google/protobuf/internal/api_implementation.py:154: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
2020-03-13T23:27:23.5253215Z     from google.protobuf.pyext import _message
2020-03-13T23:27:23.5253465Z 
2020-03-13T23:27:23.5253932Z venv/lib/python3.7/site-packages/aioesphomeapi/model.py:13
2020-03-13T23:27:23.5254500Z venv/lib/python3.7/site-packages/aioesphomeapi/model.py:13
2020-03-13T23:27:23.5255455Z   /__w/1/s/venv/lib/python3.7/site-packages/aioesphomeapi/model.py:13: DeprecationWarning: The usage of `cmp` is deprecated and will be removed on or after 2021-06-01.  Please use `eq` and `order` instead.
2020-03-13T23:27:23.5256153Z     @attr.s(cmp=True)
2020-03-13T23:27:23.5256357Z 
2020-03-13T23:27:23.5256808Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5257328Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5257870Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5258389Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5258927Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5259455Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5259980Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5260520Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5261045Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5261582Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5263055Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5263660Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5264224Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5264830Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5265415Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5266091Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5266675Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5267238Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5267819Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5268382Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5268962Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5269680Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5270270Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5270856Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5271419Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5271997Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5272566Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5273141Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5273713Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5274276Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5274857Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5275518Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5276055Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5276576Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5277112Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5277636Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5278160Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5278696Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5279223Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5279767Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5280291Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5280830Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5281359Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5282113Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5282653Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5283176Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5283660Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5284456Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5285016Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5285537Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5286070Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5286587Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5287105Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5287755Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5288280Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5288825Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5289346Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5289883Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5290558Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5291108Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5291634Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5292202Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5292747Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5293271Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5293809Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5294332Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5294871Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5295395Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5296014Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5296552Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5297076Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5297614Z venv/lib/python3.7/site-packages/reparser.py:91
2020-03-13T23:27:23.5298415Z   /__w/1/s/venv/lib/python3.7/site-packages/reparser.py:91: DeprecationWarning: Flags not at the start of the expression '(?P<md_bi1_start>(?:' (truncated)
2020-03-13T23:27:23.5299289Z     return re.compile('|'.join(patterns), re.DOTALL)
2020-03-13T23:27:23.5299659Z 
2020-03-13T23:27:23.5300174Z venv/lib/python3.7/site-packages/future/standard_library/__init__.py:65
2020-03-13T23:27:23.5300825Z venv/lib/python3.7/site-packages/future/standard_library/__init__.py:65
2020-03-13T23:27:23.5301791Z   /__w/1/s/venv/lib/python3.7/site-packages/future/standard_library/__init__.py:65: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
2020-03-13T23:27:23.5302411Z     import imp
2020-03-13T23:27:23.5302610Z 
2020-03-13T23:27:23.5303094Z venv/lib/python3.7/site-packages/feedparser/encodings.py:244
2020-03-13T23:27:23.5304075Z   /__w/1/s/venv/lib/python3.7/site-packages/feedparser/encodings.py:244: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
2020-03-13T23:27:23.5304774Z     if isinstance(proposed_encoding, collections.Callable):
2020-03-13T23:27:23.5305044Z 
2020-03-13T23:27:23.5305495Z -- Docs: https://docs.pytest.org/en/latest/warnings.html
2020-03-13T23:27:23.5306145Z ----------------- generated xml file: /__w/1/s/test-output.xml -----------------
2020-03-13T23:27:23.5306484Z 
2020-03-13T23:27:23.5306986Z ----------- coverage: platform linux, python 3.7.0-final-0 -----------
2020-03-13T23:27:23.5307378Z Coverage HTML written to dir htmlcov
2020-03-13T23:27:23.5307709Z Coverage XML written to file /__w/1/s/coverage.xml
2020-03-13T23:27:23.5307984Z 
2020-03-13T23:27:23.5308301Z ========================== slowest 10 test durations ===========================
2020-03-13T23:27:23.5308841Z 9.35s teardown tests/components/homematicip_cloud/test_init.py::test_config_with_accesspoint_passed_to_config_entry[pyloop]
2020-03-13T23:27:23.5309402Z 9.00s call     tests/components/homematicip_cloud/test_config_flow.py::test_flow_works[pyloop]
2020-03-13T23:27:23.5309904Z 5.91s teardown tests/components/default_config/test_init.py::test_setup[pyloop]
2020-03-13T23:27:23.5310423Z 2.46s call     tests/components/owntracks/test_device_tracker.py::test_mobile_multiple_async_enter_exit[pyloop]
2020-03-13T23:27:23.5310975Z 2.05s call     tests/components/verisure/test_lock.py::test_verisure_default_code[pyloop]
2020-03-13T23:27:23.5311577Z 1.51s call     tests/auth/providers/test_homeassistant.py::test_legacy_saving_loading[pyloop]
2020-03-13T23:27:23.5312102Z 1.47s teardown tests/components/gdacs/test_config_flow.py::test_step_import[pyloop]
2020-03-13T23:27:23.5312523Z 1.40s teardown tests/components/gdacs/test_config_flow.py::test_step_user[pyloop]
2020-03-13T23:27:23.5312958Z 1.31s call     tests/components/garmin_connect/test_config_flow.py::test_step_user[pyloop]
2020-03-13T23:27:23.5313402Z 1.30s teardown tests/components/homematicip_cloud/test_config_flow.py::test_flow_works[pyloop]
2020-03-13T23:27:23.5313698Z 
2020-03-13T23:27:23.5314919Z ##[error]Bash exited with code '1'.
```

## Additional information

--- a/tests/components/homematicip_cloud/conftest.py
+++ b/tests/components/homematicip_cloud/conftest.py
@@ -1,5 +1,5 @@
 """Initializer helpers for HomematicIP fake server."""
-from asynctest import CoroutineMock, MagicMock, Mock
+from asynctest import CoroutineMock, MagicMock, Mock, patch
 from homematicip.aio.auth import AsyncAuth
 from homematicip.aio.connection import AsyncConnection
 from homematicip.aio.home import AsyncHome
@@ -106,9 +106,10 @@ async def mock_hap_with_service_fixture(
 
 
 @pytest.fixture(name="simple_mock_home")
-def simple_mock_home_fixture() -> AsyncHome:
-    """Return a simple AsyncHome Mock."""
-    return Mock(
+def simple_mock_home_fixture():
+    """Return a simple mocked connection."""
+
+    mock_home = Mock(
         spec=AsyncHome,
         name="Demo",
         devices=[],
@@ -119,6 +120,27 @@ def simple_mock_home_fixture() -> AsyncHome:
         dutyCycle=88,
         connected=True,
     )
+
+    with patch(
+        "homeassistant.components.homematicip_cloud.hap.AsyncHome",
+        autospec=True,
+        return_value=mock_home,
+    ):
+        yield
+
+
+@pytest.fixture(name="mock_connection_init")
+def mock_connection_init_fixture():
+    """Return a simple mocked connection."""
+
+    with patch(
+        "homeassistant.components.homematicip_cloud.hap.AsyncHome.init",
+        return_value=None,
+    ), patch(
+        "homeassistant.components.homematicip_cloud.hap.AsyncAuth.init",
+        return_value=None,
+    ):
+        yield
 
 
 @pytest.fixture(name="simple_mock_auth")

--- a/tests/components/homematicip_cloud/test_config_flow.py
+++ b/tests/components/homematicip_cloud/test_config_flow.py
@@ -16,12 +16,15 @@ DEFAULT_CONFIG = {HMIPC_HAPID: "ABC123", HMIPC_PIN: "123", HMIPC_NAME: "hmip"}
 IMPORT_CONFIG = {HMIPC_HAPID: "ABC123", HMIPC_AUTHTOKEN: "123", HMIPC_NAME: "hmip"}
 
 
-async def test_flow_works(hass):
+async def test_flow_works(hass, simple_mock_home):
     """Test config flow."""
 
     with patch(
         "homeassistant.components.homematicip_cloud.hap.HomematicipAuth.async_checkbutton",
         return_value=False,
+    ), patch(
+        "homeassistant.components.homematicip_cloud.hap.HomematicipAuth.get_auth",
+        return_value=True,
     ):
         result = await hass.config_entries.flow.async_init(
             HMIPC_DOMAIN, context={"source": "user"}, data=DEFAULT_CONFIG
@@ -137,7 +140,7 @@ async def test_init_already_configured(hass):
     assert result["reason"] == "already_configured"
 
 
-async def test_import_config(hass):
+async def test_import_config(hass, simple_mock_home):
     """Test importing a host with an existing config file."""
     with patch(
         "homeassistant.components.homematicip_cloud.hap.HomematicipAuth.async_checkbutton",

--- a/tests/components/homematicip_cloud/test_hap.py
+++ b/tests/components/homematicip_cloud/test_hap.py
@@ -125,14 +125,11 @@ async def test_hap_create(hass, hmip_config_entry, simple_mock_home):
     hass.config.components.add(HMIPC_DOMAIN)
     hap = HomematicipHAP(hass, hmip_config_entry)
     assert hap
-    with patch(
-        "homeassistant.components.homematicip_cloud.hap.AsyncHome",
-        return_value=simple_mock_home,
-    ), patch.object(hap, "async_connect"):
+    with patch.object(hap, "async_connect"):
         assert await hap.async_setup()
 
 
-async def test_hap_create_exception(hass, hmip_config_entry):
+async def test_hap_create_exception(hass, hmip_config_entry, mock_connection_init):
     """Mock AsyncHome to execute get_hap."""
     hass.config.components.add(HMIPC_DOMAIN)
 

--- a/tests/components/homematicip_cloud/test_init.py
+++ b/tests/components/homematicip_cloud/test_init.py
@@ -24,7 +24,9 @@ from homeassistant.setup import async_setup_component
 from tests.common import MockConfigEntry
 
 
-async def test_config_with_accesspoint_passed_to_config_entry(hass):
+async def test_config_with_accesspoint_passed_to_config_entry(
+    hass, mock_connection, simple_mock_home
+):
     """Test that config for a accesspoint are loaded via config entry."""
 
     entry_config = {
@@ -51,7 +53,9 @@ async def test_config_with_accesspoint_passed_to_config_entry(hass):
     assert isinstance(hass.data[HMIPC_DOMAIN]["ABC123"], HomematicipHAP)
 
 
-async def test_config_already_registered_not_passed_to_config_entry(hass):
+async def test_config_already_registered_not_passed_to_config_entry(
+    hass, simple_mock_home
+):
     """Test that an already registered accesspoint does not get imported."""
 
     mock_config = {HMIPC_AUTHTOKEN: "123", HMIPC_HAPID: "ABC123", HMIPC_NAME: "name"}
@@ -87,7 +91,9 @@ async def test_config_already_registered_not_passed_to_config_entry(hass):
     assert config_entries[0].unique_id == "ABC123"
 
 
-async def test_load_entry_fails_due_to_connection_error(hass, hmip_config_entry):
+async def test_load_entry_fails_due_to_connection_error(
+    hass, hmip_config_entry, mock_connection_init
+):
     """Test load entry fails due to connection error."""
     hmip_config_entry.add_to_hass(hass)
 
@@ -101,7 +107,9 @@ async def test_load_entry_fails_due_to_connection_error(hass, hmip_config_entry)
     assert hmip_config_entry.state == ENTRY_STATE_SETUP_RETRY
 
 
-async def test_load_entry_fails_due_to_generic_exception(hass, hmip_config_entry):
+async def test_load_entry_fails_due_to_generic_exception(
+    hass, hmip_config_entry, simple_mock_home
+):
     """Test load entry fails due to generic exception."""
     hmip_config_entry.add_to_hass(hass)
 

